PandaSecurity/aether_endpoint_security_api/trigger_security_events.py

```diff
@@ -94,8 +33,6 @@ def _fetch_events(self) -> None:
                     break
 
             if message_batch:
-                INCOMING_MESSAGES.labels(type=event_type_name).inc(len(message_batch))
-
```

Comment:
Why do you not keep the metrics here?

Reply:
> Metrics, without intake_key, are hardly exploitable. And, as this trigger is a trigger (and not connector), we don't have this information.

We did the same for [Vade M365](https://github.com/SEKOIA-IO/automation-library/pull/1155/files#diff-17773c5f3ad8d2d54efd3afb6ab18130154a7ff34f426a18032475308ef6f768L214)